arch/arm64/boot/dts/qcom/sdm630-nokia-pl2.dts

```diff
@@ -0,0 +1,807 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/*
+ * Copyright (c) 2022, Alexander Richards
+ */
+
+/dts-v1/;
+
+#include "sdm630.dtsi"
+#include "pm660.dtsi"
+#include "pm660l.dtsi"
+
+/ {
+	//compatible = "nokia,pl2", "qcom,sdm630";
+	//chassis-type = "handset";
```

Comment:
why commented chassis type? imo you can leave it, remove above commented line with compatible (compatible is below)

Reply:
I messed with these things since the bootloader seems to be quite fickle about them; pretty sure the chassis-type line can stay, not sure the bootloader likes anything else but the downstream compatible though. Will check out; cant test it until the weekend though lol